src/map/script.c

```diff
@@ -9861,7 +9861,7 @@ BUILDIN_FUNC(monster)
 
 	if (script_hasdata(st, 9)) {
 		size = script_getnum(st, 9);
-		if (size > 3) {
+		if (size > SZ_BIG) {
 			ShowWarning("buildin_monster: Attempted to spawn non-existing size %d for monster class %d\n", size, class_);
```

Comment:
We don't need, the source already return ```SCRIPT_CMD_FAILURE```
https://github.com/rathena/rathena/blob/1bf5e69416c30fb13d4eadc27fced2609dd20589/src/map/script.c#L9866
https://github.com/rathena/rathena/blob/1bf5e69416c30fb13d4eadc27fced2609dd20589/src/map/script.c#L507